rustls/src/manual/defaults.rs

```diff
@@ -26,4 +26,42 @@ the implementation security will be improved.  We think this is an uncommon case
 Both provide roughly the same classical security level, but x25519 has better performance and
 it's _much_ more likely that both peers will have good quality implementations.
 
+### How can use the post-quantum-secure key exchange `X25519MLKEM768`?
+
+[`X25519MLKEM768`] -- a hybrid[^1], post-quantum-secure[^2] key exchange
+algorithm -- is available when using the aws-lc-rs provider.
+
+The `post-quantum` crate feature makes `X25519MLKEM768` the
+highest-priority key exchange algorithm.  Otherwise, it is available but
+not highest-priority.
+
+[X25519MLKEM768] is pre-standardization, but is now widely deployed,
+for example, by [Chrome] and [Cloudflare].
+
+You may see unexpected connection failures (such as [tldr.fail])
+-- [please report these to us][interop-bug].
+
+The two components of this key exchange are well regarded:
+X25519 alone is already used by default by rustls, and tends to have
+higher quality implementations than other elliptic curves.
+ML-KEM-768 was standardized by NIST in [FIPS203].
+
```

Comment:
WDYT about adding some mention that pure ML-KEM isn't in the default set? Could probably just reuse some text from 3f674313c421dcecfb1e70125b121305878eb6ba's commit message.